eng/targets/Settings.props

```diff
@@ -194,7 +194,7 @@
     <!-- C# specific settings -->
     <When Condition="'$(Language)' == 'C#'">
       <PropertyGroup>
-        <LangVersion>8.0</LangVersion>
+        <LangVersion>preview</LangVersion>
```

Comment:
Be careful with this, as any use of the preview features would block the ability for people to dogfood with older VS builds...